packages/scoped-elements/src/context.js

```diff
@@ -0,0 +1,16 @@
+// @ts-nocheck
+if (!window.ScopedElements) {
```

Comment:
why is this on window?

do we want to support duplicate versions of `scoped-elements`? I don't think that would be a good idea 🙈 

so if `scoped-elements` needs to be unique then this can be a local variables rigth?

Reply:
There will be almost guaranteed multiple versions of `scoped-elements` on a page. One for each bundle.

Reply:
@LarsDenBakker pointed correctly that the file could be evaluated several times in case of:
1. An application based on micro frontends with separate builds. In that case each build is going to have it's own copy of the file inside it's bundle.
2. Different versions of `@open-wc/scoped-elements` package while not all teams upgrade to the last version (this case is less probable, but possible).

Reply:
fair enough - so it is by "design" like that...

then it should definitely be a more unique name - maybe `window.__scopedElements`?

and I guess it would be smart to add the major version number as well e.g. so we can later do backwards compatibility checks ala

```js
let scopedElements = window.__scopedElements
if (window.__scopedElements.majorVersion === 0) {
  scopedElements = upgradeIfNeeded(window.__scopedElements);
}

// then always use scopedElements
```

Reply:
If the file is evaluated several times then the cache is going to be splitted into different caches and elements are going to be registered several times with different tag names. Probably is not a big issue but not desirable, don't you think?

Reply:
On the other hand, with local variables is going to work also and code is cleaner...

Reply:
Yeah I'm unsure whether it's a problem or not, not sure if a large custom element registry is a problem? We could also start without a global variable and change it later if needed?

Reply:
Simpler is better. We could refactor it later.

Reply:
fixed it